bindings/rust/bench/README.md

```diff
@@ -4,12 +4,14 @@ We use to Criterion.rs to benchmark s2n-tls against two commonly used TLS librar
 
 ## Setup 
 
-Setup is easy! Just have OpenSSL installed and generate Rust bindings for s2n-tls using `bindings/rust/generate.sh`.
+Setup is easy! Just have OpenSSL installed and generate Rust bindings for s2n-tls using `bindings/rust/generate.sh`. To bench with AWS-LC, Amazon's custom libcrypto implementation, run `use-awslc-s2n.sh` or `use-awslc-rustls.sh`, which sets up the benchmarks to use AWS-LC for s2n-tls and Rustls, respectively. To stop benching with AWS-LC, delete `.cargo/config.toml`.
```

Comment:
I think there is also an assumption that we are running on a certain platform. Can you add which platform you have been testing on and where its expected to work. Possibly a started command to install all the dependencies